### DISCUSSION TITLE
Multiplayer Tactical RPG

### DISCUSSION THREAD

**Discussion by E-A-N** [enable3d/enable3d#180] [Show and tell]
<img src="https://cdn.discordapp.com/attachments/831407650456469515/859179234865053766/movementStats.gif" game gif> </img>

Building a multiplayer focused game that's similar to Final Fantasy Tactics.  The 1st prototype will be in Phaser/Enabled3D then I'll probably rebuild in Godot after the game gets funded.

**Comment by yandeu** [comment]
Nice! Would love to test it 🤩

**Reply by E-A-N** [reply]
Sure no problem, it'll probably be ready for pre-alpha testing in 1-2 months.
